### PR TITLE
playbook that provisions figgy staging

### PR DESCRIPTION
**Pull request by kayiwa**
this is an attempt to fix the notify on slack feature that gives us a
simpler playbook for figgy staging

### CODE REVIEW

File: playbooks/figgy_staging.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+- hosts: figgy
```
Comment (hackmastera) [inline playbooks/figgy_staging.yml]: I think we should change it in hosts, too

File: playbooks/figgy_staging.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (hackmastera) [inline playbooks/figgy_staging.yml]: delete all this from the figgy playbook

**Comment by jrgriffiniii** [comment]
See #401

File: playbooks/figgy_staging.yml (hunk 2 above)
Comment (hackmastera) [reply playbooks/figgy_staging.yml]: rename that playbook `figgy_production`

**Comment by jrgriffiniii** [comment]
I'm still getting the same error:
```
fatal: [figgy-staging1.princeton.edu -> localhost]: FAILED! => {"changed": false, "module_stderr":"sudo: a password is required\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 1}
```

This might be due some issue in my local env.

**Comment by jrgriffiniii** [comment]
I've found the problem, this is the offending line: https://github.com/pulibrary/princeton_ansible/pull/410/files#diff-7c1890847b60b189fec0fed7e1faec44R33

Attempting to escalate privileges on `localhost` requires `sudo` with a password depending upon the local environment where `ansible-playbook` is being invoked.

**Pull request merged**